FakeItEasy.sln

```diff
@@ -1,26 +1,32 @@
-﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 2013
-VisualStudioVersion = 12.0.31101.0

```

Comment:
>  The current sln on master can be opened by anyone using VS2015. Indeed, on my main dev machine I only have VS2015.

But I have VS2013 and VS2015. Up 'til now, the solution has only opened in VS2013. Today (after I checked out your PR branch) it opened in VS2015, so that's new.

A while ago, we were discussing moving to MSBuild 14 only and you came down against it, [until there was a demonstrable benefit](https://github.com/FakeItEasy/FakeItEasy/issues/537#issuecomment-177367158), such as using shared projects. That made sense to me.
So why not stick with what we have for now?

You've at least hinted that things work fine for you on your VS2015-only machine, yes? Or at least no worse than they would with this change?
And there's a potentially small degradation (or at least change) for those of us who have VS2013 and VS2015…